reflex/components/datadisplay/code.py

```diff
@@ -505,16 +523,11 @@ def _render(self):
         out = super()._render()
         predicate, qmark, value = self.theme._var_name.partition("?")
         out.add_props(
-            style=Var.create(
+            style=ImmutableVar.create(
                 format.to_camel_case(f"{predicate}{qmark}{value.replace('`', '')}"),
-                _var_is_local=False,
-                _var_is_string=False,
-            )
-        ).remove_props("theme", "code")
-        if self.code is not None:
-            out.special_props.add(
-                Var.create_safe(f"children={str(self.code)}", _var_is_string=False)
             )
+        ).remove_props("theme", "code").add_props(children=self.code)
```

Comment:
overall much nicer!

is is possible to get rid of the `format.to_camel_case(f"{predicate}{qmark}{value.replace('`', '')}")` and just use `self.theme` directly?